src/components/TimeFrameControl.tsx

```diff
@@ -172,12 +171,7 @@ const TimeFrameControl: FC<Props> = ({
     />
   ) : (
     <div className="flex flex-row flex-wrap items-center lg:gap-x-1">
-      {(version === "all"
```

Comment:
Ah great, you've cleaned this up as well 🙏 .

Reply:
As usual, deleted code is the best code😅